src/main/java/net/greatstart/dto/DtoInterest.java

```diff
@@ -0,0 +1,24 @@
+package net.greatstart.dto;
+
+
+import lombok.Data;
+import net.greatstart.model.Category;
+import org.springframework.format.annotation.NumberFormat;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.validation.constraints.Size'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)